src/DssSpell.sol

```diff
@@ -69,202 +126,182 @@ contract DssSpellAction is DssAction {
+    address internal immutable MKR_SKY                  = DssExecLib.getChangelogAddress("MKR_SKY");
+    address internal immutable USDS                     = DssExecLib.getChangelogAddress("USDS");
+    address internal immutable ALLOCATOR_ROLES          = DssExecLib.getChangelogAddress("ALLOCATOR_ROLES");
+    address internal immutable ALLOCATOR_NOVA_A_VAULT   = DssExecLib.getChangelogAddress("ALLOCATOR_NOVA_A_VAULT");
+    address internal immutable ALLOCATOR_NOVA_A_BUFFER  = DssExecLib.getChangelogAddress("ALLOCATOR_NOVA_A_BUFFER");
+    address internal immutable ALLOCATOR_BLOOM_A_VAULT  = DssExecLib.getChangelogAddress("ALLOCATOR_BLOOM_A_VAULT");
+    address internal immutable ALLOCATOR_SPARK_A_VAULT  = DssExecLib.getChangelogAddress("ALLOCATOR_SPARK_A_VAULT");
+    address internal constant NOVA_OPERATOR             = 0x0f72935f6de6C54Ce8056FD040d4Ddb012B7cd54;
 
     // ---------- Wallets ----------
-    address internal constant LIQUIDITY_BOOTSTRAPPING = 0xD8507ef0A59f37d15B5D7b630FA6EEa40CE4AFdD;
-    address internal constant ECOSYSTEM_TEAM = 0x05F471262d15EECA4059DadE070e5BEd509a4e73;
+    address internal constant BLUE           = 0xb6C09680D822F162449cdFB8248a7D3FC26Ec9Bf;
+    address internal constant BONAPUBLICA    = 0x167c1a762B08D7e78dbF8f24e5C3f1Ab415021D3;
+    address internal constant CLOAKY_2       = 0x9244F47D70587Fa2329B89B6f503022b63Ad54A5;
+    address internal constant WBC            = 0xeBcE83e491947aDB1396Ee7E55d3c81414fB0D47;
+    address internal constant TANGO          = 0xB2B86A130B1EC101e4Aed9a88502E08995760307;
+    address internal constant AEGIS_D        = 0x78C180CF113Fe4845C325f44648b6567BC79d6E0;
+    address internal constant CLOAKY_KOHLA_2 = 0x73dFC091Ad77c03F2809204fCF03C0b9dccf8c7a;
+    address internal constant SKY_STAKING    = 0x05c73AE49fF0ec654496bF4008d73274a919cB5C;
+
+    // ---------- Grove Proxy ----------
+    // Note: The deployment address of the Grove Proxy can be found at https://forum.sky.money/t/technical-scope-of-the-star-2-allocator-launch/26190
+    address internal constant GROVE_PROXY = 0x1369f7b2b38c76B6478c0f0E66D94923421891Ba;
+
+    // ---------- Nova Proxy ----------
+    // Note: The deployment address of the Nova Proxy can be found at https://forum.sky.money/t/technical-scope-of-the-nova-allocator-adjustment/27175
+    address internal constant NOVA_PROXY = 0x355CD90Ecb1b409Fdf8b64c4473C3B858dA2c310;
 
     // ---------- Spark Proxy Spell ----------
     // Note: Spark Proxy: https://github.com/sparkdotfi/sparklend-deployments/blob/bba4c57d54deb6a14490b897c12a949aa035a99b/script/output/1/primary-sce-latest.json#L2
     address internal constant SPARK_PROXY = 0x3300f198988e4C9C63F75dF86De36421f06af8c4;
-    address internal constant SPARK_SPELL = 0xe7782847eF825FF37662Ef2F426f2D8c5D904121;
+    address internal constant SPARK_SPELL = 0x7B28F4Bdd7208fe80916EBC58611Eb72Fb6A09Ed;
```

Comment:
Confirming this address matches what has been reviewed and tested by the Phoenix Labs team [here](https://github.com/sparkdotfi/spark-spells/pull/115/).

Reply:
Confirming this is the correct address .